master/getting-started/mesos/tutorials/connecting-tasks.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Connecting to Tasks
+---
+
+Calico assigns each mesos task a uniquely addressable IP address.
+In testing, it is easy to test connectivity by making requests directly to this
+IP.
+
+However, when designing a multi-tier application, it is not usually sensible to
+hardcode applications to statically connect to these addresses directly, as they are dynamic
+and often change as the tasks die and are reinstantiated across the cluster.
+
+Instead, it is advisable to program the applications to connect to one another
+using DNS entries which dynamically resolve to the applications' current addresses.
```

Comment:
Why not skip most of this and just say

"Since the IP addresses are dynamically assigned and may change when a task is re-deployed, it is recommended to use one of the available service discovery solutions available for Mesos.